src/instructlab/sdg/filterblock.py

```diff
@@ -69,8 +100,10 @@ def __init__(
         super().__init__(ctx, block_name)
         self.value = filter_value if isinstance(filter_value, list) else [filter_value]
         self.column_name = filter_column
-        self.operation = operation
-        self.convert_dtype = convert_dtype
+        self.operation = _get_operator_func(operation)
+        self.convert_dtype = _get_convert_dtype(convert_dtype)
+        if self.convert_dtype:
+            self.value = [self.convert_dtype(value) for value in self.value]
```

Comment:
This changed the type of the parameters passed to the constructor. I will push a commit to update the docstring that is just above this

Reply:
https://github.com/instructlab/sdg/pull/86/commits/0c4cdc20e0140e87e9c1e4e892b18f2d1c157d7e